content/social_coding.md

```diff
@@ -138,14 +138,28 @@ When you find a repository with code you would like to reuse, you may look at th
 > ... am I allowed to use it?
 - Contribution guide 
 > ... how to contribute and decision process?
+- Code of Conduct
+> ... how to make clear which behaviors are unacceptable and discouraged? How violations of Code of Conduct will be handled?
 - Trust and community 
 > ... somebody you trust recommended it?
 
 
 ... most of which you have or will learn during this [CodeRefinery](https://coderefinery.org) workshop!
 
+```{challenge} Exercises (15 mn)
+Let's go back to the repository `recipe` we have created at the beginning og this CodeRefinery workshop. This repository contains a guacamole recipe and we would like you to think about what to add in your repository to improve its reusability and encourage external contributions. Using the list given above, pick the items (one or two) you think are the most important. 
```

Comment:
...beginning **of** this...
...what you could add...?

Reply:
Thanks for spotting all my spelling errors (I thought they would be spotted by CI!). I also change the second sentence with your suggestion.